WTWSMS/common/minor_titles/00_minor_titles.txt

```diff
@@ -2646,7 +2597,10 @@ title_royal_builder = {
 	}
 
 	allowed_to_grant = {
-		is_feudal = yes
+		OR = {
+			is_feudal = yes
+			is_merchant_republic = yes
+		}
```

Comment:
@loup99 

SHOOT, I didn't make the connection that Merchant Republics aren't royalty. I confused "nobility" with "royalty" again. I'll fix that immediately.